web/cobrands/fixmystreet/assets.js

```diff
@@ -485,7 +485,8 @@ function get_asset_pick_message() {
     var message;
     if (typeof this.fixmystreet.asset_item_message !== 'undefined') {
         message = this.fixmystreet.asset_item_message;
-        message = message.replace('ITEM', this.fixmystreet.asset_item);
+        var re = new RegExp('ITEM', 'g');
+        message = message.replace(re, this.fixmystreet.asset_item);
```

Comment:
This is fine, but note that `replace(/ITEM/g, ...)` would work just as well :)

Reply:
I couldn't find anything definitive about browser support for that notation so decided to play it safe.

Reply:
It's the same :)